aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,63 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 2023
+
+- **Brand Refresh**
+  - [New brand refresh is now available on Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=0%3A1&t=ViKCtdnRM2CUkNZI-1)!
+  - To view details on how to solve for common Figma problems or to migrate to the new theme, check out the [Figma migration guide](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A4&t=ViKCtdnRM2CUkNZI-1).
+  - [Wizard titles](https://design-system.hpe.design/templates/wizard) have gone from bold to medium font weight.
+- **Figma Component Updates**
+ - Figma components have been updated to use the correct theme colors for placeholder and disabled text. Previously, the placeholder text was xweak. It has been updated to use the correct color, text-weak to meet accessibility. The following components have been updated:
```

Comment:
Looks a little strange to have the "Figma Component Updates" and "Figma components have been updated..." at the same bullet point level
<img width="780" alt="Screen Shot 2023-04-26 at 4 03 50 PM" src="https://user-images.githubusercontent.com/54560994/234713094-91985c3f-05c4-4270-afa9-88b76f2a93e4.png">

Reply:
I had trouble removing the bullet point so it wouldn't look like this. Can you help with that? :)